tests/python_tests/test_llm_pipeline.py

```diff
@@ -586,7 +586,8 @@ def run_perf_metrics_collection(model_descr, generation_config: Dict, prompt: st
 def test_perf_metrics(model_descr, generation_config, prompt):
     import time
     start_time = time.perf_counter()
-    perf_metrics = run_perf_metrics_collection(read_model(model_descr), generation_config, prompt)
+    # To ensure the prefill stage takes much more time make initial prompt long.
+    perf_metrics = run_perf_metrics_collection(read_model(model_descr), generation_config, prompt * 200)
     total_time = (time.perf_counter() - start_time) * 1000
```

Comment:
in general, assumptions about perf metrics like `assert load_time < 1000.0` should not be in tests, because depending on machine / network stability `load_time` may significantly vary